pkg/utils/bpf-iter-ns/bpf-iter-ns.go

```diff
@@ -55,11 +57,26 @@ func ReadOnCurrentPidNs(iter *link.Iter) ([]byte, error) {
 		return nil, fmt.Errorf("open BPF iterator: %w", err)
 	}
 	defer file.Close()
-	buf, err := io.ReadAll(file)
-	if err != nil {
-		return nil, fmt.Errorf("read BPF iterator: %w", err)
+
+	var b bytes.Buffer
+	r := bufio.NewReader(file)
+	buf := make([]byte, 8192)
```

Comment:
Could you add a comment somewhere to explain why we can't use io.ReadAll (with the relevant link to the Linux commit)?